extensions/ql-vscode/src/queryserver-client.ts

```diff
@@ -171,6 +171,22 @@ export class QueryServerClient extends DisposableObject {
       args.push('--old-eval-stats');
     }
 
+    if (await this.cliServer.cliConstraints.supportsStructuredEvalLog()) {
+      if (this.config.structuredEvalLogFile) {
+        args.push('--evaluator-log');
+        args.push(this.config.structuredEvalLogFile);
+  
+        if (this.config.structuredEvalLogMinify) {
+          args.push('--evaluator-log-minify');
+        }
+        // 1 is default behavior in query server if argument is not passed.
+        if (this.config.structuredEvalLogVerbosity > 1) {
```

Comment:
Should you throw an error if the value is > 5?

Is there a way to specify the 1 < x < 5 constraint in the setting itself?

Reply:
Just did both: specified the constraint in the setting (as part of `package.json`) and ignored the value if it wasn't within bounds!

Reply:
We decided to hard-code this to highest verbosity for now (and anticipate this to be the same for the per-query logs).